SU2_CFD/src/output/CFlowIncOutput.cpp

```diff
@@ -295,7 +296,7 @@ void CFlowIncOutput::SetVolumeOutputFields(CConfig *config){
   AddVolumeOutput("VELOCITY-Y", "Velocity_y", "SOLUTION", "y-component of the velocity vector");
   if (nDim == 3)
     AddVolumeOutput("VELOCITY-Z", "Velocity_z", "SOLUTION", "z-component of the velocity vector");
-  if (heat || weakly_coupled_heat)
+  if (heat || weakly_coupled_heat || flamelet)
```

Comment:
Is the flamelet model turning on the energy equation implicitly? Instead can you enforce in the config postprocessing that the energy equation is on for flamelets? That way the logic doesn't spread

Reply:
No, the energy equation is off, we switch it off in cconfig. We instead solve an equation for the total enthalpy (sensible + chemical). We look up the temperature from the lookup table as a function of total enthalpy and progress-variable and we put that temperature in the temperature field. That is why we still want to write this field to file.